packages/components/src/button/style.scss

```diff
@@ -138,9 +138,13 @@
 		white-space: nowrap;
 		color: $components-color-accent;
 		background: transparent;
+		font-weight: $font-weight-medium;
+		-webkit-font-smoothing: antialiased;
 
 		&:hover:not(:disabled, [aria-disabled="true"]) {
-			box-shadow: inset 0 0 0 $border-width $components-color-accent-darker-10;
+			box-shadow: inset 0 0 0 $border-width $components-color-accent-darker-20;
+			color: $components-color-accent-darker-20;
+			background: rgba(var(--wp-admin-theme-color--rgb), 0.04);
```

Comment:
I think we'll eventually get to this when it comes to component theming, ideally we shouldn't be using transparency at all because it doesn't account for different surface colors. 

That said, I'm happy to make any adjustments here if you think it's worth while.

Reply:
```suggestion
			background: color-mix(in srgb, $components-color-accent 4%, transparent);
```